niworkflows/utils/spaces.py

```diff
@@ -476,13 +490,16 @@ def references(self):
     @property
     def cached(self):
```

Comment:
If we're okay with fiddling with the API still, I would suggest changing this to `cache`. Intuitively, `obj.cached` feels like it should be a bool, while `obj.cache` would return the cache.

Reply:
I don't feel too strongly about either way, only minor pain I see is changing all examples / upstream calls. If we decide to go that route, let's do it on a separate PR

Reply:
Sure.